src/ontodia/widgets/toolbar.tsx

```diff
@@ -73,6 +77,33 @@ export class EditorToolbar extends React.Component<Props, void> {
         );
     }
 
+    private renderButtonsTogglePanels = () => {
+        const {
+            isViewOnly,
+            isLeftPanelOpen,
+            onLeftPanelToggle,
+            isRightPanelOpen,
+            onRightPanelToggle
+        } = this.props;
+
+        if (isViewOnly) { return null; }
+
+        const className = `ontodia-btn ontodia-btn-default ${CLASS_NAME}__toggle`;
+
+        return (
+            <div className='ontodia-btn-group ontodia-btn-group-sm'>
+                <button type='button'
+                        className={`${className} ${CLASS_NAME}__toggle--left ${isLeftPanelOpen ? 'active' : ''}`}
```

Comment:
It would be better to use BEM style for flag-like classes:

 * `${CLASS_NAME}__toggle-left` for left toggle button,
 * `${CLASS_NAME}__toggle-left--active` when button is active,
 * same for right panel.